src/get-throw.js

```diff
@@ -0,0 +1,22 @@
+
+export default function getThrow() {
+    //Gen random number
+    let randomNumber = Math.random() * 3;
+
+    //call getThrowFromNumber
+    const toss = getThrowFromNumber(randomNumber);
+    //spit out result
+    return toss;
+}
+
+export function getThrowFromNumber(num) {
+
+    if(num < 1) {
+        return 'rock';
+    } else if(num < 2) {
+        return 'paper';
+    } else { return 'scissors'; }
+
```

Comment:
watch out for any unnecessary white space - since it isn't an issue with the rest of your code I'm not going to take points off, but in the future that's something I'm supposed to dock for!